tests/lib/reset.sh

```diff
@@ -119,7 +119,7 @@ reset_all_snap() {
                 ;;
             *)
                 # make sure snapd is running before we attempt to remove snaps, in case a test stopped it
-                if ! systemctl status snapd.service snapd.socket; then
+                if ! systemctl status snapd.service snapd.socket >/dev/null; then
```

Comment:
We should probably be using `systemctl is-active ...` but this is good enough

Reply:
+1 but I think it's absent on old systemd version.